### ISSUE TITLE
Request Martian Wars Classic, Motor Wars Wasteland, Air Wars 1 and Motor Toons to the portal

### ISSUE THREAD

**Issue by kiskofa2006** [jjaquinta/martian-games-portal#46]

**Comment by kiskofa2006** [comment]
*adding those game to the portal

**Comment by kiskofa2006** [comment]
And Motor Wars 1 too, i forget that

**Comment by jjaquinta** [comment]
Can you please specify what database these games are likely linked to?

**Comment by kiskofa2006** [comment]
Yes, https://martiangames.com/games/MotorToons https://martiangames.com/games/MotorWars https://martiangames.com/games/AirWars and Motor Wars Wasteland and Martian Wars only existing in the database, that you showed previously, is it possible to add those 2 games from there?

**Comment by jjaquinta** [comment]
What I need is the names of the databases. Then I can examine them and see if they are compatible or not.

**Comment by kiskofa2006** [comment]
If you mean that ypu sent me earelier, then: motor toons is "MotorToons", air wars is "AirWars", Martian Wars possibly "MoonWars", motor wars wasteland possibly "MotorWars3" and Motor Wars is "motorwars"
